### PR TITLE
Feature - Use pandas dataframes

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Use `DataFrame`s instead of `json` loading, which is more hadny and data-science like.

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description
Example of saved DataFrame:
```
{"title":"Change primary link to adviser docs","body":"","size":"XS","created_by":"fridex","created_at":1617185147,"closed_at":null,"closed_by":null,"merged_at":null,"commits_number":1,"id":"11"}
{"title":"API is no longer internal","body":"","size":"XS","created_by":"fridex","created_at":1617041250,"closed_at":1617092758,"closed_by":"fridex","merged_at":1617092758,"commits_number":1,"id":"10"}
{"title":"Remove archived thoth-package-analyzer","body":"","size":"XS","created_by":"fridex","created_at":1612858098,"closed_at":1615450479,"closed_by":"goern","merged_at":1615450479,"commits_number":1,"id":"9"}
{"title":"Rename link to justifications","body":"","size":"XS","created_by":"fridex","created_at":1600165045,"closed_at":1600165228,"closed_by":"fridex","merged_at":1600165228,"commits_number":1,"id":"8"}
{"title":"Add a link to YouTube channel","body":"","size":"XS","created_by":"fridex","created_at":1588648761,"closed_at":1588653047,"closed_by":"goern","merged_at":1588653047,"commits_number":1,"id":"7"}
{"title":"Add a link to AICoE Python package index","body":null,"size":"XS","created_by":"fridex","created_at":1576759858,"closed_at":1588648973,"closed_by":"fridex","merged_at":1588648973,"commits_number":1,"id":"6"}
{"title":"Change link name - use docs","body":null,"size":"XS","created_by":"fridex","created_at":1575892549,"closed_at":1576759779,"closed_by":"fridex","merged_at":1576759779,"commits_number":2,"id":"5"}
{"title":"Add a link to Thamos docs","body":null,"size":"XS","created_by":"fridex","created_at":1575892373,"closed_at":1576759790,"closed_by":"fridex","merged_at":1576759790,"commits_number":1,"id":"4"}
{"title":"Add build-analyzers and package-analyzer to developer's menu","body":"Related: https:\/\/github.com\/thoth-station\/build-analyzers\/pull\/37\r\nRelated: https:\/\/github.com\/thoth-station\/package-analyzer\/pull\/26\r\nRelated: https:\/\/github.com\/thoth-station\/nepthys\/pull\/14\r\n","size":"L","created_by":"fridex","created_at":1564472959,"closed_at":1564482506,"closed_by":"harshad16","merged_at":1564482506,"commits_number":2,"id":"3"}
{"title":"Provide link to Kebechet in menu","body":null,"size":"XS","created_by":"fridex","created_at":1556880114,"closed_at":1557135920,"closed_by":"goern","merged_at":1557135920,"commits_number":1,"id":"2"}
{"title":"State how to update changes","body":"","size":"XS","created_by":"fridex","created_at":1554713637,"closed_at":1555307166,"closed_by":"goern","merged_at":1555307166,"commits_number":1,"id":"1"}
```
Example of its Hive scheme:
```
DROP TABLE thoth_srcops.test_json;
CREATE TABLE thoth_srcops.test_json (
  title STRING,
  body STRING,
  size STRING,
  created_by STRING,
  created_at TIMESTAMP,
  closed_at TIMESTAMP,
  closed_by STRING,
  merged_at TIMESTAMP,
  commits_number INTEGER,
  github_id INTEGER
)
ROW FORMAT SERDE 'org.openx.data.jsonserde.JsonSerDe'
LOCATION 's3a://DH-PLAYPEN/dtuchyna/srcopsmetrics/bot_knowledge/test_json/';
```
Hive table screenshot:
image

### CODE REVIEW

**Comment by xtuchyna** [comment]
/hold
not yet complete

**Comment by xtuchyna** [comment]
Hello @pacospace , could you please review this?

File: srcopsmetrics/entities/pull_request.py [hunk 1]
```diff
@@ -94,11 +98,11 @@ def store(self, pull_request: GithubPullRequest):
             "closed_by": closed_by,
             "merged_at": merged_at,
             "commits_number": commits,
-            "referenced_issues": PullRequest.get_referenced_issues(pull_request),
-            "interactions": GitHubKnowledge.get_interactions(pull_request.get_issue_comments()),
-            "reviews": self.extract_pull_request_reviews(pull_request),
-            "requested_reviewers": self.extract_pull_request_review_requests(pull_request),
-            "labels": GitHubKnowledge.get_labels(pull_request.as_issue()),
+            # "referenced_issues": PullRequest.get_referenced_issues(pull_request),
```
Comment (pacospace) [inline srcopsmetrics/entities/pull_request.py:101]: why these are commented?

**Review by pacospace — COMMENTED** [review]
One q, otherwise lgtm

File: srcopsmetrics/entities/pull_request.py (hunk 1 above)
Comment (xtuchyna) [reply srcopsmetrics/entities/pull_request.py:101]: nice catch! these features are being changed in #377 , so I can revert changes back here.

**Review by pacospace — APPROVED** [review]
lgtm!

**Pull request merged**